### DISCUSSION TITLE
Test de la serrure connectée Tedee Go + accessoires - Haade.fr

### DISCUSSION THREAD

**Discussion by giscus** [haade-administrator/haade.fr#41] [Comments]
# Test de la serrure connectée Tedee Go + accessoires - Haade.fr

Test complet et integration dans home-assistant de la serrure Tedee Go avec le bridge wifi Tedee et le clavier code accès Tedee

https://haade.fr/fr/blog/test-de-la-serrure-connectee-tedee-go-et-des-accessoires

**Comment by sebastienserre** [comment]
Le bruit est vraiment le point noir de ces apareils. J'ai une Nuki 3Pro et c'est également très bruyant.

**Reply by haade-administrator** [reply]
Bonjour, c'est clair que tu ne l'activera pas en pleine nuit, et ils sont pas près d'y remédier , j'ai fait des suggestions en plaçant le moteur au centre ou en l'isolant
